config/services/freeipa-trust.xml

```diff
@@ -9,6 +9,6 @@
   <port protocol="udp" port="389"/>
   <port protocol="tcp" port="445"/>
   <port protocol="udp" port="445"/>
-  <port protocol="tcp" port="1024-1300"/>
+  <port protocol="tcp" port="49152-65535"/><!-- Dynamic RPC Ports -->
```

Comment:
Would it be better to add a `samba-rpc` service definition and then include, see 1fc208bf93171920196ed771b6a4c709ce6607aa, that from there and `samba-dc` ? `freeipa-trust` has a lot of overlap with `samba-dc`.

Reply:
FreeIPA and Samba AD are different enough, it is better to keep the port definitions for them separated.

For this single item I do not see a need to create a separate definition -- unless you have a way to define a virtual service that could be overridden by an administrator and default to something otherwise. These port ranges depend on what is defined on AD side -- if AD administrators set a different range for their DCs, Linux administrators would need to change this and Samba configuration in FreeIPA.

Reply:
ACK. Good enough for me.